astropy/timeseries/periodograms/base.py

```diff
@@ -1,6 +1,9 @@
 import abc
+
 import numpy as np
-from astropy.timeseries import TimeSeries, BinnedTimeSeries
+
+from astropy.timeseries.binned import BinnedTimeSeries
```

Comment:
Huh, why would isort complain about this one?

Reply:
`TimeSeries` wasn't used and astropy is first-party, so it's separated from third-party.

Reply:
No, I mean why it made you go from `from astropy.timeseries import BinnedTimeSeries` to `from astropy.timeseries.binned import BinnedTimeSeries`?

Reply:
Oh. That was a huge pain. time series has an import cycle that's preventable by importing directly from the source.